src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const firstThreeCalls = 3;
+
+  return (firstClosure) => {
+    count++;
+
+    return (secondClosure) => {
+      if (count > firstThreeCalls && count % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return firstClosure + secondClosure;
```

Comment:
You can switch "if" to ternary operator and your code will be much shorter and in my opinion more professional)